tests/system/conftest.py

```diff
@@ -57,6 +57,14 @@ def not_postgres(database_dialect):
         )
 
 
+@pytest.fixture(scope="session")
+def not_google_standard_sql(database_dialect):
+    if database_dialect == DatabaseDialect.GOOGLE_STANDARD_SQL:
+        pytest.skip(
+            f"{_helpers.DATABASE_DIALECT_ENVVAR} set to GOOGLE_STANDARD_SQL in environment."
+        )
```

Comment:
I'm not 100% sure about this fixture. It's for a session, but if I'm not mistaken, it still needs to be used by a test to be actually executed. Like `not_postgres()` fixture, for example - it's the same, but it's called by several tests (by passing the fixture name as an argument of a test). But this one seems to be not used anywhere, thus, never executed.

Reply:
Hey Ilya, so i will be using this when I introduce jsonb functionality which is only supported in postgres